app_dart/lib/src/request_handlers/deflake_flaky_builders.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:github/github.dart';
+import 'package:meta/meta.dart';
+import 'package:yaml/yaml.dart';
+
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../service/bigquery.dart';
+import '../service/config.dart';
+import '../service/github_service.dart';
+import 'flaky_handler_utils.dart';
+
+@immutable
+class DeflakeFlakyBuilders extends ApiRequestHandler<Body> {
```

Comment:
Can we add docs for classes and functions? Also for earlier added two APIs? I believe they would be helpful for whoever will work on these APIs later.

Reply:
added